src/SFA.DAS.AssessorService.Web/Controllers/AccountController.cs

```diff
@@ -232,21 +232,13 @@ public IActionResult UpdateAnAccount()
         [HttpGet]
         public IActionResult CreateAnAccount()
         {
-            if (_config.UseGovSignIn)
-            {
-                return RedirectToAction("UpdateAnAccount");
-            }
-            var vm = new CreateAccountViewModel();
-            return View(vm);
+            return RedirectToAction("UpdateAnAccount");
         }
 
         [HttpPost]
         public async Task<IActionResult> CreateAnAccount(CreateAccountViewModel vm)
-        {
-            if (_config.UseGovSignIn)
```

Comment:
The code after the redirect will never be executed, and posting back now always redirects to an error - how does the post get called - it would never be a good idea to post back now - can it be posted back - if so then need to work out what to do with that code

Reply:
If the Post CreateAnAccount method simply redirects to an error page and the CreateAnAccount view calls the Post method, then neither the Post method nor the view are required anymore. Would you agree?

I think the Get CreateAnAccount method is called by the "back" button on the InviteSent view. Since that always now redirects to the Get UpdateAnAccount method, surely it would make sense to replace the view's call with a direct one and delete the Get Create method from the controller?

Are there any cases where we still manage identities internally? I can see there are some code paths from the InviteSent process which don't obviously depend on the GovSignIn variable. The InviteSent action calls upon the CreateAccountViewModel. What is the purpose of the InviteSent process? Would it make sense to rename the view model to "Update..." rather than "Create..." or do you think we may be able to delete the InviteSent code paths?

I can't see anywhere where the Get InviteSent method is called. Is it called by any external processes, do you think?